tasks/upgrade.yml

```diff
@@ -45,6 +45,7 @@
       - description: "Upgrading host: {{ item.name }}"
 
 - name: Gather self-heal facts about all gluster hosts in the cluster
+  connection: ssh
```

Comment:
Could you please move `connection: ssh` to line 59 right after `delegate_to`, so it's more obvious those options are connected? And could you please also add a comment that this is needed because of https://github.com/ansible/ansible/issues/70184 ?

Reply:
Addressed